<ui version="4.0" >
 <class>UIPluginsSettingsElement</class>
 <widget class="QWidget" name="UIPluginsSettingsElement" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Plugins Settings Element</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QCheckBox" name="cbEnabled" >
       <property name="toolTip" >
        <string>Check this to enable/disable the plugin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lIcon" >
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="scaledContents" >
        <bool>false</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTitle" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
       <property name="indent" >
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSettings" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Settings</string>
       </property>
       <property name="icon" >
        <iconset resource="../../resources/resources.qrc" >
         <normaloff>:/edit/icons/edit/settings.png</normaloff>:/edit/icons/edit/settings.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAbout" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>About</string>
       </property>
       <property name="icon" >
        <iconset resource="../../resources/resources.qrc" >
         <normaloff>:/help/icons/help/qt.png</normaloff>:/help/icons/help/qt.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="lDescription" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <widget class="QCheckBox" name="cbNeverEnable" >
     <property name="toolTip" >
      <string>Check this for forcing the plugin to never be activated</string>
     </property>
     <property name="text" >
      <string>Never enable this plugin</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
